public/pages/CreateTrigger/containers/ConfigureActions/ConfigureActions.js

```diff
@@ -378,7 +378,7 @@ class ConfigureActions extends React.Component {
           hasShadow={!flyoutMode}
           hasBorder={!flyoutMode}
         >
-          {!flyoutMode && !loadingDestinations && numActions < 1 ? (
+          {!flyoutMode && loadingDestinations && numActions < 1 ? (
```

Comment:
This may cause issues as I don't believe the component will always have `loadingDestinations` set to `true`; that should be set to `false` when the `getChannels` API finishes execution. If you have the notifications plugin installed, and attempt to configure some actions for a trigger, does the component function as expected?

Reply:
It does get set to false on line 179 and inside the catch blocks.

Reply:
Ah, gotcha. Thanks for pointing that out.